docs/architecture/runtime-agent-contract.md

```diff
@@ -15,9 +15,17 @@ Runtime Agent automates the process of [establishing trusted connection](./estab
 
 Depending on the type of trusted connection, during the runtime lifecycle, it may be required to periodically [renew trusted connection](./establishing-trusted-connection.md#client-certificate-flow---certificate-renewal).
 
-## Configuring the runtime
+## Configuring the Runtime
 
-Runtime Agent periodically requests for the configuration of its Runtime from the Management Plane. Changes in the configuration for the Runtime are applied by the Runtime Agent on the Runtime. To fetch the Runtime configuration the Runtime Agent calls `applicationsForRuntime(runtimeId: ID!, first: Int = 100, after: PageCursor)` query offered by Director. The response for the query contains a page with list of Applications assigned for the Runtime and info about next page. Each Application will contain only credentials that are valid for the runtime that called the query. Each Runtime Agent can fetch the configurations for Runtimes that belong to its tenant, there is no validation if the Runtime Agent is fetching the configuration for the runtime on which it runs.
+Runtime configuration is connecting Applications to Runtime, which are situated in the same scenario(s).
```

Comment:
This sentence is a bit confusing. How does this connection process look like? I think this should be rephrased or removed.